tools/devops/automation/build-cronjob.yml

```diff
@@ -7,124 +7,38 @@
   default: true
 
-- name: runDeviceTests
-  displayName: Run Device Tests 
-  type: boolean
-  default: false
-
-- name: runOldMacOSTests
-  displayName: Run Tests on older macOS versions 
-  type: boolean
-  default: true
+- name: stageDisplayNamePrefix
+  type: string
+  default: ''
 
-- name: runWindowsIntegration
-  displayName: Run Windows integration tests
-  type: boolean
-  default: true
+- name: dependsOn
+  type: string
+  default: ''
 
-- name: runGovernanceTests
-  displayName: Run Governance Checks
-  type: boolean
-  default: true
+- name: dependsOnResult
+  type: string
+  default: ''
 
-- name: runSamples
-  displayName: Run Samples
+- name: isPR
   type: boolean
   default: false
-  
-- name: enableDotnet
-  displayName: Build Dotnet 
-  type: boolean
-  default: true
 
-- name: enableAPIDiff
-  displayName: Enable API diff generation
-  type: boolean
-  default: true
+- name: repositoryAlias
+  type: string
+  default: self
 
-- name: forceInsertion
-  displayName: Force Insertion 
-  type: boolean
-  default: false 
+- name: commit
+  type: string
+  default: HEAD
 
-- name: skipESRP
-  displayName: Skip ESRP
-  type: boolean
-  default: false # only to be used when testing the CI and we do not need a signed pkg
-
-- name: simTestsConfigurations
-  displayName: Simulator test configurations to run
-  type: object
-  default: []
-
-- name: deviceTestsConfigurations
-  displayName: Device test configurations to run
-  type: object
-  default: [
-    {
-      testPrefix: 'iOS64',
-      stageName: 'ios64b_device',
-      displayName: 'iOS64 Device Tests',
-      testPool: 'VSEng-Xamarin-Mac-Devices',
-      useXamarinStorage: false,
-      testsLabels: '--label=run-ios-64-tests,run-non-monotouch-tests,run-monotouch-tests,run-mscorlib-tests',
-      statusContext: 'VSTS: device tests iOS',
-      makeTarget: 'vsts-device-tests',
-      extraBotDemands: [
-        'ios',
-      ]
-    },
-    {
-      testPrefix: 'tvos',
-      stageName: 'tvos_device',
-      displayName: 'tvOS Device Tests',
-      testPool: 'VSEng-Xamarin-Mac-Devices',
-      useXamarinStorage: false,
-      testsLabels: '--label=run-tvos-tests,run-non-monotouch-tests,run-monotouch-tests,run-mscorlib-tests',
-      statusContext: 'VSTS: device tests tvOS',
-      makeTarget: 'vsts-device-tests',
-      extraBotDemands: [
-        'tvos',
-      ]
-    }]
-
-- name: macTestsConfigurations
-  displayName: macOS test configurations to run
-  type: object
-  default: [
-    {
-      stageName: 'mac_11_5_m1',
-      displayName: 'M1 - Mac Big Sur (11.5)',
-      macPool: 'VSEng-VSMac-Xamarin-Shared',
-      useImage: false,
-      statusContext: 'M1 - Mac Big Sur (11.5)',
-      demands: [
-        "Agent.OS -equals Darwin",
-        "macOS.Name -equals BigSur",
-        "macOS.Architecture -equals arm64",
-        "Agent.HasDevices -equals False",
-        "Agent.IsPaired -equals False"
-      ]
-    }]
+variables:
+- name: System.Debug
+  value: true
```

Comment:
AFAIK this is going to set everything to debug, do you want that?

Reply:
Good call, may be helpful in case there are any issues this first run, but I can add debugging later if necessary!